docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -93,7 +93,7 @@ This sample creates the following data in the current worksheet:
 
```

Comment:
Should the cells/ranges include file go in this article? I think it's a good intro concept that ties into the object model nicely.